src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -1273,6 +1304,14 @@ private static DkmClrCompilationResultFlags GetLocalResultFlags(LocalSymbol loca
 
                 switch (fieldKind)
                 {
+                    case GeneratedNameKind.AnonymousTypeField:
+                        Debug.Assert(fieldName == field.Name); // This only happens once.
+                        fieldName = fieldName.Substring(openBracketOffset + 1, closeBracketOffset - openBracketOffset - 1);
+                        goto REPARSE;
+                    case GeneratedNameKind.TransparentIdentifier:
+                        // A transparent identifier in a transparent identifier.
```

Comment:
Actually, that's not this case.  Should this read "A transparent identifier in an anonymous type?".